gridt/src/app/home/home.page.html

```diff
@@ -59,9 +59,9 @@ <h3>
                         </h3>
                         <p>{{ isLeaderDone(leader, movement) ? 'Done!' : 'Pending...' }}</p>
                       </div>
-                      <small *ngIf="isLeaderDone(leader, movement) && leader.last_signal.message">
-                        {{ leader.last_signal.message }}
-                      </small>
+                      <div *ngIf="isLeaderDone(leader, movement) && leader.last_signal.message" id="expandable">
+                        <app-expandable signalMessage={{leader.last_signal.message}}></app-expandable>
```

Comment:
I would like there to be a different name here

Reply:
Is leadersMessage a better name?

Reply:
Resolved via what's app.